[2025-01-31T02:59:02.790+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-31T02:59:02.813+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: validate_and_insert_csv_to_postgres_with_branch_adjusted.check_table_branch_task manual__2025-01-31T02:58:58.428605+00:00 [queued]>
[2025-01-31T02:59:02.830+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: validate_and_insert_csv_to_postgres_with_branch_adjusted.check_table_branch_task manual__2025-01-31T02:58:58.428605+00:00 [queued]>
[2025-01-31T02:59:02.834+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-01-31T02:59:02.858+0000] {taskinstance.py:2888} INFO - Executing <Task(BranchPythonOperator): check_table_branch_task> on 2025-01-31 02:58:58.428605+00:00
[2025-01-31T02:59:02.874+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=164) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-31T02:59:02.876+0000] {standard_task_runner.py:72} INFO - Started process 166 to run task
[2025-01-31T02:59:02.876+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'validate_and_insert_csv_to_postgres_with_branch_adjusted', 'check_table_branch_task', 'manual__2025-01-31T02:58:58.428605+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/read_file_updated.py', '--cfg-path', '/tmp/tmpmdctpkj5']
[2025-01-31T02:59:02.877+0000] {standard_task_runner.py:105} INFO - Job 9: Subtask check_table_branch_task
[2025-01-31T02:59:02.942+0000] {task_command.py:467} INFO - Running <TaskInstance: validate_and_insert_csv_to_postgres_with_branch_adjusted.check_table_branch_task manual__2025-01-31T02:58:58.428605+00:00 [running]> on host e2498a6c8083
[2025-01-31T02:59:03.026+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='validate_and_insert_csv_to_postgres_with_branch_adjusted' AIRFLOW_CTX_TASK_ID='check_table_branch_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-31T02:58:58.428605+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-31T02:58:58.428605+00:00'
[2025-01-31T02:59:03.028+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-31T02:59:03.029+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-31T02:59:03.029+0000] {logging_mixin.py:190} INFO - Current task name:check_table_branch_task state:running start_date:2025-01-31 02:59:02.815906+00:00
[2025-01-31T02:59:03.030+0000] {logging_mixin.py:190} INFO - Dag name:validate_and_insert_csv_to_postgres_with_branch_adjusted and current dag run status:running
[2025-01-31T02:59:03.030+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-31T02:59:03.032+0000] {baseoperator.py:405} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-01-31T02:59:03.040+0000] {base.py:84} INFO - Retrieving connection 'postgres_default'
[2025-01-31T02:59:03.051+0000] {logging_mixin.py:190} INFO - Employee Data table exists. Proceeding to check files.
[2025-01-31T02:59:03.052+0000] {python.py:240} INFO - Done. Returned value was: check_files_branch_task
[2025-01-31T02:59:03.053+0000] {branch.py:38} INFO - Branch into check_files_branch_task
[2025-01-31T02:59:03.054+0000] {skipmixin.py:230} INFO - Following branch ('check_files_branch_task',)
[2025-01-31T02:59:03.064+0000] {skipmixin.py:278} INFO - Skipping tasks [('create_table_employee_data_task', -1)]
[2025-01-31T02:59:03.100+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-31T02:59:03.101+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=validate_and_insert_csv_to_postgres_with_branch_adjusted, task_id=check_table_branch_task, run_id=manual__2025-01-31T02:58:58.428605+00:00, execution_date=20250131T025858, start_date=20250131T025902, end_date=20250131T025903
[2025-01-31T02:59:03.109+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-31T02:59:03.110+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-31T02:59:03.111+0000] {logging_mixin.py:190} INFO - Dag name:validate_and_insert_csv_to_postgres_with_branch_adjusted queued_at:2025-01-31 02:58:58.501774+00:00
[2025-01-31T02:59:03.112+0000] {logging_mixin.py:190} INFO - Task hostname:e2498a6c8083 operator:BranchPythonOperator
[2025-01-31T02:59:03.134+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-31T02:59:03.161+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-31T02:59:03.166+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
